src/main/java/com/cdancy/jenkins/rest/JenkinsClient.java

```diff
@@ -172,6 +186,18 @@ public Builder overrides(final Properties overrides) {
             return this;
         }
 
+        /**
+         * Optional List of Module to add. Modules can be added, for logging
+         * for example.
+         *
+         * @param modules optional List of Module to add.
+         * @return this Builder.
+         */
+        public Builder modules(final List<Module> modules) {
+            this.modules = modules;
+            return this;
+        }
```

Comment:
Lets re-write this method like so:
```
public Builder modules(final Module ...modules) {
    this.modules.addAll(modules)
    return this
}
```
This will allow the caller to invoke the builder method multiple times should they choose.